gradle.properties

```diff
@@ -1,29 +1,40 @@
-# IntelliJ Platform Artifacts ReplistModel.size < i || i >= 0ositories -> https://plugins.jetbrains.com/docs/intellij/intellij-artifacts.html
+# IntelliJ Platform Artifacts Repositories -> https://plugins.jetbrains.com/docs/intellij/intellij-artifacts.html
+
 pluginGroup=com.redhat.devtools.intellij
 pluginName=LSP4IJ
 pluginRepositoryUrl=https://github.com/redhat-developer/lsp4ij
 # SemVer format -> https://semver.org
-# NO SPACES AROUND THE EQUALS SIGN!!
 pluginVersion=0.10.0-SNAPSHOT
+
 # Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
-pluginSinceBuild=232
-#pluginUntilBuild=233.*
+pluginSinceBuild=233
+#pluginUntilBuild=243.*
+
 # IntelliJ Platform Properties -> https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#configuration-intellij-extension
 platformType=IC
-platformVersion=2023.2
+
```

Comment:
for now, please still use the same version, then we'ell update to some new version later on.